site/content/en/docs/Getting Started/create-fleetautoscaler.md

```diff
@@ -256,8 +256,14 @@ simple-udp-mzhrl-zg9rq   Ready     10.30.64.99    7745    minikube     5m
 ```
 
 {{< alert title="Note" color="info">}}
-If you want to update a `Fleet`, which has `RollingUpdate` replacement strategy and is controlled by a `FleetAutoscaler`, you could omit `Replicas` parameter in a `Fleet` Spec.
-Otherwise fleet would be scaled according to Fleet `Replicas` parameter first and only after certain amount of time it would be rescaled to fit `FleetAutoscaler` `BufferSize` parameter.
+If you want to update a `Fleet` which has `RollingUpdate` replacement strategy and is controlled by a `FleetAutoscaler`:
+1. With `kubectl apply`: you should omit `replicas` parameter in a `Fleet` Spec before creating this `Fleet`.
+1. With `kubectl edit`: you should not change `replicas` parameter in a `Fleet` Spec when updating other field parameter.
+
+If you follow both rules above, then `maxSurge` and `maxUnavaiable` parameters would be used and update would actually be a RollingUpdate.
+Otherwise fleet would be scaled according to Fleet `replicas` parameter first and only after certain amount of time it would be rescaled to fit `FleetAutoscaler` `BufferSize` parameter.
```

Comment:
```suggestion
Otherwise the Fleet would be scaled according to Fleet `replicas` parameter first and only after certain amount of time it would be rescaled to fit `FleetAutoscaler` `BufferSize` parameter.
```